unindent test fails with "ambiguous name" when tested as a standalone crate

<!--  ❤️  -->
```
~/explore
❯ cargo download -x unindent
INFO: cargo-download v0.1.2
INFO: Latest version of crate unindent=* is 0.2.0
INFO: Crate `unindent==0.2.0` downloaded successfully
INFO: Crate content extracted to ./unindent-0.2.0/
~/explore
❯ cd unindent-0.2.0/
~/explore/unindent-0.2.0
❯ cargo t
   Compiling unindent v0.2.0 (/Users/i/explore/unindent-0.2.0)
    Finished test [unoptimized + debuginfo] target(s) in 0.37s
     Running unittests src/lib.rs (target/debug/deps/unindent-d72c7e1565632189)

running 0 tests

test result: ok. 0 passed; 0 failed; 0 ignored; 0 measured; 0 filtered out; finished in 0.00s

   Doc-tests unindent
error[E0659]: `unindent` is ambiguous
  --> /Users/i/explore/unindent-0.2.0/src/lib.rs:59:9
   |
59 | pub use unindent::*;
   |         ^^^^^^^^ ambiguous name
   |
   = note: ambiguous because of multiple potential import sources
   = note: `unindent` could refer to a crate passed with `--extern`
   = help: use `::unindent` to refer to this crate unambiguously
note: `unindent` could also refer to the module defined here
  --> /Users/i/explore/unindent-0.2.0/src/lib.rs:57:1
   |
57 | mod unindent;
   | ^^^^^^^^^^^^^
   = help: use `crate::unindent` to refer to this module unambiguously

error: aborting due to previous error

For more information about this error, try `rustc --explain E0659`.
error: doctest failed, to rerun pass `--doc`

Caused by:
  process didn't exit successfully: `rustdoc --edition=2021 --crate-type lib --crate-name unindent --test /Users/i/explore/unindent-0.2.0/src/lib.rs -L dependency=/Users/i/explore/unindent-0.2.0/target/debug/deps -L dependency=/Users/i/explore/unindent-0.2.0/target/debug/deps --extern unindent=/Users/i/explore/unindent-0.2.0/target/debug/deps/libunindent-2485fc53a8f836b8.rlib -C embed-bitcode=no --error-format human` (exit status: 1)
```

--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -30,7 +30,7 @@ jobs:
       - uses: dtolnay/rust-toolchain@master
         with:
           toolchain: ${{matrix.rust}}
-      - run: cargo test
+      - run: cargo test --workspace
 
   minimal:
     name: Minimal versions

--- a/unindent/src/lib.rs
+++ b/unindent/src/lib.rs
@@ -56,4 +56,4 @@
 
 mod unindent;
 
-pub use unindent::*;
+pub use crate::unindent::*;

